builtin/files/stack-templates/network.json.tmpl

```diff
@@ -471,6 +471,36 @@
       "Type": "AWS::EC2::SecurityGroupIngress"
     },
     {{ end }}
+    {{if ((index .PluginConfigs "kube2iam").Enabled) }}
```

Comment:
By assuming the port number as 8181, you could create conflict with other users who are using that same port for other applications, and other people may not need to open a hole in the worker's security groups in order to scrape that endpoint.